spec.md

```diff
@@ -519,7 +519,7 @@ class Talker
 
 ### 4.3 Properties and Constants
 
-Visibility MUST be declared on all properties.
+Visibility MUST be declared on all properties.  If set-visibility is specified, then the general visibility MAY be omitted.
```

Comment:
```suggestion
Visibility MUST be declared on all properties. If set-visibility is specified, then the general visibility MAY be omitted.
```

Double spaces after a period aren't rendered by markdown unless they are followed by a newline. Most editors agree that two spaces after a period doesn't improve readability past the printed era.